### DISCUSSION TITLE
Display flashed with Gotcha DWIN_SET but no changes on display?

### DISCUSSION THREAD

**Discussion by weav2k4** [Jyers/Marlin#1311] [General Discussion]
I am using an 8gb fat32 card and copied the Gotcha version of DWIN_SET.  I renamed it to be just "DWIN_SET".  When flashing the display after about 3 seconds it does go orange indicating it succeeded.  Turned off / remove card / power back up and I do not have any of the Gotcha's icons displayed.  Not sure what I'm missing?  I am using the precompiled 1.3.5 on my v2.  Any suggestions?

**Comment by Cal-LF** [comment]
Try to reformat the SD card using the long method in windows. The screen is finicky about the SD card. It should be working with your procedure.
I have an 8gb that is the only one that will work. 16 and 32 gb don't work even if partitioned to 8gb.

**Reply by weav2k4** [reply]
Finally after 4 different cards one of them worked!  Thanks for the help.

**Reply by tfaz099** [reply]
I cannot find a way to format the sd card on a MAC, do you have any suggestions or can point to a flexible formatting program for a mac ?
thank you

**Reply by tfaz099** [reply]
I used the 8gig card that came with the printer... worked fine.  any differences with the Gotcha firmware than the Jyers? or just different look?

**Reply by weav2k4** [reply]
It's just a different look is all.

**Comment by GeoStork** [comment]
I've updated my scree with the Gotcha pack and I don't believe I see a difference so I may have an issue and just need to keep trying.  Could someone post a quick picture of a screen with new icons that I can compare to?  Thanks.

**Comment by Cal-LF** [comment]
![Image01](https://user-images.githubusercontent.com/76499230/130230431-9dc0f9eb-385a-4a3c-8519-bee854911128.jpg)

**Reply by GeoStork** [reply]
Thanks.  I'm not having any success with the SD cards that I have.

**Reply by weav2k4** [reply]
What size card you using?  I had to use a 8GB card and had to format it FAT32 allocated with 4k not the default 32k to get it to work.

**Reply by galea101** [reply]
yes i can confirm this, after trying 6 different card all below 8mb. i changed 1 at random to 4k instead of 32k and it flash screen first time. thanks weav2k4.

**Comment by GeoStork** [comment]
Yes, that worked!  Thank you!

**Comment by GeoStork** [comment]
Sadly no, I cannot.  I had to “punt” and use a PC to format an 8gb card to the correct allocation size.  Google searching seemed to say that you could do this on the Mac BUT you need to use UNIX commands and the Terminal program.  Too geeky for me.

**Comment by tfaz099** [comment]
Thank you

I have a windows puter also so maybe that’ll work.  Does it need to use Jyers as the base firmware or is it completely stand alone on the display ?

I am ok with the command line but rather it be more simple

Thanks again,
tony

“Once you choose hope, anything’s possible”


On 8/29/21, 9:45 PM, "GeoStork" ***@***.******@***.***>> wrote:

Sadly no, I cannot. I had to “punt” and use a PC to format an 8gb card to the correct allocation size. Google searching seemed to say that you could do this on the Mac BUT you need to use UNIX commands and the Terminal program. Too geeky for me.



> On Aug 29, 2021, at 9:41 PM, tfaz099 ***@***.***> wrote:
>
>
> I cannot find a way to format the sd card on a MAC, do you have any suggestions or can point to a flexible formatting program for a mac ?
> thank you
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/Jyers/Marlin/discussions/1311#discussioncomment-1251637>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ASYBTN2AO7GE52JETFQ62XTT7LOVLANCNFSM5CFLMKMQ>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Jyers/Marlin/discussions/1311#discussioncomment-1251644>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATDLIPYRSNC5ASEVBB4RYK3T7LPDLANCNFSM5CFLMKMQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by GeoStork** [comment]
I believe the icons are independent of Jyers firmware. For one thing they reside on the display eprom and I say that because he references a Github project (repository?) where the icons are developed and there are tools where you can make your own.  In other words, they are created to work on the DWIN display type like the V2 uses.

Doug

**Comment by tfaz099** [comment]
Ok, so i gave it a go.
First i updated my LCD... all good
Then i opened up Jyers in my VS editor and there are a bunch of config repositories.
OK so i got it compiled and uncommented #define BABYSTEP_ALWAYS_AVAILABLE
no matter what i do babystepping all the time is not working.
The "live adjustment" checkbox seems useless as it needs to re-home "z" and if i am adjusting my first layer and wanting to babystep it up or down it simply does not work. am i missing something ?
It works fine in Smith3D's last release but cannot get it working here. Anyway to get rid of that entire "live adjustment" checkbox... i cannot see where i would ever use it in the way it is presented.
Thank you
tony



“Once you choose hope, anything’s possible”


On 8/29/21, 10:12 PM, "GeoStork" ***@***.******@***.***>> wrote:

I believe the icons are independent of Jyers firmware. For one thing they reside on the display eprom and I say that because he references a Github project (repository?) where the icons are developed and there are tools where you can make your own. In other words, they are created to work on the DWIN display type like the V2 uses.

Doug

> On Aug 29, 2021, at 10:02 PM, tfaz099 ***@***.***> wrote:
>
>
>
> Thank you
>
> I have a windows puter also so maybe that’ll work. Does it need to use Jyers as the base firmware or is it completely stand alone on the display ?
>
> I am ok with the command line but rather it be more simple
>
> Thanks again,
> tony
>
> “Once you choose hope, anything’s possible”
>
>
> On 8/29/21, 9:45 PM, "GeoStork" ***@***.******@***.***>> wrote:
>
> Sadly no, I cannot. I had to “punt” and use a PC to format an 8gb card to the correct allocation size. Google searching seemed to say that you could do this on the Mac BUT you need to use UNIX commands and the Terminal program. Too geeky for me.
>
>
>
> > On Aug 29, 2021, at 9:41 PM, tfaz099 ***@***.***> wrote:
> >
> >
> > I cannot find a way to format the sd card on a MAC, do you have any suggestions or can point to a flexible formatting program for a mac ?
> > thank you
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub <https://github.com/Jyers/Marlin/discussions/1311#discussioncomment-1251637>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ASYBTN2AO7GE52JETFQ62XTT7LOVLANCNFSM5CFLMKMQ>.
> > Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub<https://github.com/Jyers/Marlin/discussions/1311#discussioncomment-1251644>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATDLIPYRSNC5ASEVBB4RYK3T7LPDLANCNFSM5CFLMKMQ>.
> Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/Jyers/Marlin/discussions/1311#discussioncomment-1251673>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ASYBTN5OBZXVMWZ5BWH7K7DT7LRDNANCNFSM5CFLMKMQ>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Jyers/Marlin/discussions/1311#discussioncomment-1251690>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATDLIP433U3M74JIWSB5NT3T7LSJZANCNFSM5CFLMKMQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by unixneo** [reply]
You can (re)format the SD card with Disk Utilities on macOS.  I had the same problem today with the UI firmware upgrade on my Ender 3v2 and reformatted the SD card and the magic trick worked.
